docker-compose.yml

```
@@ -116,6 +118,30 @@ services:
       - prometheus
     networks:
       - nestjs-network
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)